src/MobileList.js

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from 'react';
+import moment from 'moment';
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+require('twix');
```

Comment:
Is this rando ‘require’ intentional?

Reply:
cc: @john-osullivan 

Hey John, can you review this?